internal/gatewayapi/translator_test.go

```diff
@@ -507,7 +507,7 @@ func TestTranslateWithExtensionKinds(t *testing.T) {
 
 func overrideOutputConfig(t *testing.T, data string, filepath string) {
 	t.Helper()
-	file, err := os.OpenFile(filepath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0o755)
+	file, err := os.OpenFile(filepath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0o644)
```

Comment:
should we change this in a separate PR? It causes a lot of file changes.

Reply:
This change didn't cause the file changes. I explicitly `chmod`-ed the yaml files which were marked as executable so that they wouldn't be.

This change only makes sure that new files being added are created without the executable bit set.